### ISSUE TITLE
Permissions

### ISSUE THREAD

**Issue by vulture64** [thelinuxchoice/saycheese#4]
Is there any way to force the permission to "allow"" camera permission?

**Comment by swagkarna** [comment]
no victim has to allow permissions

**Comment by minanagehsalalma** [comment]
@swagkarna  how to change the page content ?

**Comment by vik01** [comment]
So what's the use of this then? If victim has to allow media access

**Comment by minanagehsalalma** [comment]
> So what's the use of this then? If victim has to allow media access

A pop that asks the user to allow media then when he does it sends you the pics automatically. .  Is this clear enough  ?

**Comment by thomastesla-acdc** [comment]
> > So what's the use of this then? If victim has to allow media access
> 
> A pop that asks the user to allow media then when he does it sends you the pics automatically. . Is this clear enough ?
The purpose of the thing is unclear still

**Comment by minanagehsalalma** [comment]
> > A pop that asks the user to allow media then when he does it sends you the pics automatically. . Is this clear enough ?
> 
> The purpose of this thing is unclear still

Nah it's clear 
Let me ask you  a question .... Can you do the mentioned above manually ? 
If the answer is no  then this the purpose of the tool  .

**Comment by Rockyd845** [comment]
@minanagehsalalma hey buddy but i don't think that anybody will allow camera permissions

**Comment by minanagehsalalma** [comment]
> @minanagehsalalma hey buddy but i don't think that anybody will allow camera permissions

@Rockyd845 but you can get them to click on it accidentally. ..and this where you need to use  your brain in.

**Comment by Rockyd845** [comment]
@minanagehsalalma that means you found the way for this....can u plz share

**Comment by minanagehsalalma** [comment]
> that means you found the way for this....can u plz share

@Rockyd845 not really. ...it's up to you and your smartness and social engineering skill.
